views/includes/date_of_birth-display.html

```diff
@@ -1,6 +1,9 @@
-{% if dob_year %}
-{% set dob = dateMacros.formatDate(dob_day, dob_month, dob_year) %}
+{% if date_of_birth["day"] == "01" or dob_day == "01"%}
```

Comment:
" %}

Reply:
do we need to check  {%if date_of_birth["month"]}?
just because it is done on line 5 below

Reply:
the check for month would cause dob to be undefined and the UI to just say "born". So of to leave as is or remove check on line 5.